test/test_response.py

```diff
@@ -448,6 +448,49 @@ def test_decode_zstd_incomplete(self, data: bytes) -> None:
         with pytest.raises(DecodeError):
             HTTPResponse(fp, headers={"content-encoding": "zstd"})
 
+    decode_param_set = [
+        b"f",
+        b"fo",
+        b"foo",
+        b"foo" * 5,
+        b"123456789012345",
+        b"foo4",
+        b"x" * 100,
+    ]
```

Comment:
It doesn't look like the bigger number of cases is more useful, let's use the ones from `test_decode_zstd_incomplete`

Reply:
```
        b"foo" * 5,
        b"123456789012345",
```

behaved differently (even if it was wrong code).

And I believe that testing for a single char should be included (b"f")

So, possibly go with this (at least for now)

```
    decode_param_set = [
        b"f",
        b"foo",
        b"foo" * 5,
        b"123456789012345",
        b"x" * 100,
    ]
```

Note that if I am to work on the read1 related issue, I would want to have a broader param_set anyways. Can be always reduced down again.

I can of course change this PR to the original set of 2, and use an extended set locally. No problem, so, if you still prefere, I can restore the set here.

Reply:
=> reduced to original set